assets/js/edit-task.js

```diff
@@ -0,0 +1,55 @@
+import { CLICK_EVENT, HIDE_CLASS } from "/assets/js/constants.js";
+
+function editTaskEvent(
+  doneButton,
+  editButton,
+  deleteButton,
+  revertButton,
+  saveButton,
+  inputElement,
+  tools,
+  allTasks,
+  id
+) {
+  editButton.addEventListener(CLICK_EVENT, () => {
+    const prevContent = inputElement.innerText;
+    inputElement.contentEditable = true;
+    inputElement.focus();
+    editButton.classList.add(HIDE_CLASS);
+    deleteButton.classList.add(HIDE_CLASS);
+    revertButton.classList.remove(HIDE_CLASS);
+    saveButton.classList.remove(HIDE_CLASS);
+
+    function doneEvent() {
```

Comment:
function name should express the intention properly, `doneEvent` doesn't represent it's handling the `done` state of a task.

```suggestion
    function onTodoDone() {
```